dbt_project.yml

```diff
@@ -22,5 +22,8 @@ clean-targets:         # directories to be removed by `dbt clean`
   - "target"
   - "dbt_packages"
 
+data_tests:
+  +store_failures: true
+
```

Comment:
Are these lines for debugging. If so, we can probably consider adding to README, but not to the project.

Reply:
data_tests - is related with the testing part
store_failures - it will crate a table into the db with these failures
I think is rather a bad thing to have them, so I will remove it.